packages/ketcher-polymer-editor-react/src/components/modal/monomerConnection/MonomerConnections.tsx

```diff
@@ -159,6 +161,15 @@ function AttachmentPointSelectionPanel({
     [bonds],
   );
 
+  const getMonomerCap = (attachmentPoint): MonomerCap => {
```

Comment:
I'm not sure about naming here. If anyone could suggest a better term, I would appreciate.

Reply:
I would suggest to rename it to 'getLeavingGroup'